www/smtube/Portfile

```diff
@@ -1,10 +1,19 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           qmake 1.0
+
+if {${os.platform} eq "darwin" && ${os.major} < 17} {
+    PortGroup       qmake 1.0
+
+} else {
+    PortGroup       qmake5 1.0
+
+    qt5.depends_component   qttools qtwebkit-examples
```

Comment:
```suggestion
    qt5.depends_component   qttools qtwebkit
```

Reply:
It seems unlikely that a build would depend on qtwebkit-examples rather than qtwebkit... what do you think?

Reply:
@kencu I believe, you are right.

I just saw the source mentions `Qt5WebKitWidgets`:
https://github.com/smplayer-dev/smtube/blob/d44edf344684ab9bb4117dd031e926b042989cd1/smtube.spec#L31
And only found this in variables: https://github.com/macports/macports-ports/blob/de80cfa1d564f5986397f76eb25ca865e1a87bd7/_resources/port1.0/group/qt5_variables-1.0.tcl#L358-L363
But it is indeed unlikely that examples will be needed. Let me try changing that.